sktime/forecasting/sarimax.py

```diff
@@ -0,0 +1,188 @@
+        :math:`a + bt + ct^3`. Default is to not include a trend component.
+    measurement_error : bool, optional, default=False
+        Whether or not to assume the endogenous observations `endog` were
+        measured with error.
+    time_varying_regression : bool, optional, default=False
+        Used when an explanatory variables, `exog`, are provided
+        to select whether or not coefficients on the exogenous regressors are
+        allowed to vary over time.
+    mle_regression : bool, optional, default=True
+        Whether or not to use estimate the regression coefficients for the
+        exogenous variables as part of maximum likelihood estimation or through
+        the Kalman filter (i.e. recursive least squares). If
+        `time_varying_regression` is True, this must be set to False.
+    simple_differencing : bool, optional, default=False
+        Whether or not to use partially conditional maximum likelihood
+        estimation. If True, differencing is performed prior to estimation,
+        which discards the first :math:`s D + d` initial rows but results in a
+        smaller state-space formulation. See the Notes section for important
+        details about interpreting results when this option is used. If False,
+        the full SARIMAX model is put in state-space form so that all
+        datapoints can be used in estimation.
+    enforce_stationarity : bool, optional, default=True
+        Whether or not to transform the AR parameters to enforce stationarity
+        in the autoregressive component of the model.
+    enforce_invertibility : bool, optional, default=True
+        Whether or not to transform the MA parameters to enforce invertibility
+        in the moving average component of the model.
+    hamilton_representation : bool, optional, default=False
+        Whether or not to use the Hamilton representation of an ARMA process
+        (if True) or the Harvey representation (if False).
+    concentrate_scale : bool, optional, default=False
+        Whether or not to concentrate the scale (variance of the error term)
+        out of the likelihood. This reduces the number of parameters estimated
+        by maximum likelihood by one, but standard errors will then not
+        be available for the scale parameter.
+    trend_offset : int, optional, default=1
+        The offset at which to start time trend values. Default is 1, so that
+        if `trend='t'` the trend is equal to 1, 2, ..., nobs. Typically is only
+        set when the model created by extending a previous dataset.
+    use_exact_diffuse : bool, optional, default=False
+        Whether or not to use exact diffuse initialization for non-stationary
+        states. Default is False (in which case approximate diffuse
+        initialization is used).
+
+    See Also
+    --------
+    sktime.forecasting.arima.ARIMA
+    sktime.forecasting.arima.AutoARIMA
+    sktime.forecasting.statsforcast.StatsForecastAutoARIMA
```

Comment:
```suggestion
    ARIMA
    AutoARIMA
    StatsForecastAutoARIMA
```
can you try this? pls check if it was build fine and if its linking

Reply:
I'm not sure what you mean by linking... These lines show up as docstring just like the documents surrounding it.

Reply:
![2022-04-09 11_06_15-sarimax py - Sktime-dev (Workspace) - Visual Studio Code](https://user-images.githubusercontent.com/55965636/162581986-0730e239-8222-496c-9bf7-47eea6e92c46.jpg)
 
I'm using VS Code. This is how they show up

Reply:
you need to check the `readthedocs` in the CI/CD

Reply:
They are rendered as text still. Is there anything special I need to do to embed a link?
![2022-04-09 12_23_28-SARIMAX — sktime documentation and 34 more pages - Personal - Microsoft​ Edge](https://user-images.githubusercontent.com/55965636/162584794-c09398fd-6c51-4704-8864-4bbf02d5ee85.jpg)

Reply:
I see the problem, but I don´t know the answer.

This seems to occur a couple other times too, in the current docs, e.g., `BATS`. I was not aware of that either.

Since it is a pre-existing problem for which we do not seem to know the solution, you would not be introducing a new one and risk spending time to sort it.

I would hence suggest, open an issue about the *pre-existing problem* and link to a few examples; and we merge this PR without addressing it. Another PR should address *all occurrences* of this problem at the same time, once we understand how to detect and address it systemically..

Reply:
Issue #2427 has been opened. A few examples of when it works and when it doesn't are included.